forms-flow-bpm/src/main/java/org/camunda/bpm/extension/hooks/listeners/TaskAssignmentListener.java

```diff
@@ -252,12 +359,22 @@ private String getSubject(DelegateExecution delegateExecution) {
         return String.valueOf(this.subject.getValue(delegateExecution));
     }
 
-    private String getRecipient(DelegateExecution delegateExecution) {
-        return String.valueOf(this.recipientEmail.getValue(delegateExecution));
+    private InternetAddress[] getRecipients(DelegateExecution delegateExecution) throws AddressException {
+        String[] recipientArray = String.valueOf(this.recipientEmails.getValue(delegateExecution)).split(",");
+        InternetAddress[] recipients = new InternetAddress[recipientArray.length];
```

Comment:
Can make this a bit cleaner by using a Java stream + map instead of using a for loop. Maybe something like this:

```java
InternetAddress[] recipients = Arrays.stream(recipientArray)
       .map(r -> InternetAddress.parse(r)[0])
       .toArray(InternetAddress[]::new)

```